sretoolbox/container/image.py

```diff
@@ -103,14 +107,16 @@ class Image:
 
     def __init__(self, url, tag_override=None, username=None, password=None,
                  auth_server=None, response_cache=None, auth_token=None,
-                 ssl_verify=True):
+                 ssl_verify=True, session=None, timeout=None):
         image_data = self._parse_image_url(url)
         self.scheme = image_data['scheme']
         self.registry = image_data['registry']
         self.repository = image_data['repository']
         self.image = image_data['image']
         self.response_cache = response_cache
         self.ssl_verify = ssl_verify
+        self.session = session
+        self.timeout = timeout
```

Comment:
You have to pass the session and the timeout to the `__getitem__` call.

Reply:
Oh, there is another `Image` usage inside `Image`! Thank you for the spot.

Reply:
fixed in https://github.com/app-sre/sretoolbox/pull/105